assignments/array-methods.js

```diff
@@ -56,28 +56,77 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 // ==== Challenge 1: Use .forEach() ====
 // The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
 let fullName = [];
+
+runners.forEach(element => {
+    fullName.push(element.first_name + " " + element.last_name);    
+});
 console.log(fullName);
 
+
+
+
 // ==== Challenge 2: Use .map() ====
 // The event director needs to have all the runner's first names converted to uppercase because the director BECAME DRUNK WITH POWER. Convert each first name into all caps and log the result
 let allCaps = [];
+
+runners.map((item) => {
+    allCaps.push(item.first_name.toUpperCase());
```

Comment:
No need to use `push` when using `.map()`, as it already creates a copy of new array with modified data from the old one, so better aproach would be something like this:

```
allCaps = runners.map(item => {
  return item.first_name.toUpperCase();
});
```